app/views/users/new.html.erb

```diff
@@ -0,0 +1,55 @@
+<h1>Create new user</h1>
+
+<!-- <form action="/users" method="post" accept-charset="UTF-8">
+  <label for="username">Username:</label>
+  <input type="text" id="username" name="user[username]"><br><br>
+
+  <label for="email">Email:</label>
+  <input type="email" id="email" name="user[email]"><br><br>
+
+  <label for="password">Password:</label>
+  <input type="password" id="password" name="user[password]"><br><br>
+
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+
+  <input type="submit" value="Create">
+  
+</form> -->
+
+<!-- <%= form_tag('/users', :method => "post") do %>
+  <%= label_tag(:username, "Username: ") %>
+  <%= text_field_tag(:username) %>
+
+  <%= label_tag(:email, "Email: ") %>
+  <%= text_field_tag(:email) %>
+
+  <%= label_tag(:password, "Password: ") %>
+  <%= text_field_tag(:password) %>
+
+  <%= submit_tag("Create") %>
+
+<% end %> -->
+
+<%= form_with model: @user, method: :post do |form| %>
+  <% @user.errors.full_messages_for(:username).each do |message| %>
+    <div><%= message %></div>
+  <% end %>
+  <% @user.errors.full_messages_for(:email).each do |message| %>
+    <div><%= message %></div>
+  <% end %>
+  <% @user.errors.full_messages_for(:password).each do |message| %>
+    <div><%= message %></div>
+  <% end %>
+  <br>
+
+  <%= form.label :username, "Username:" %><br>
+  <%= form.text_field :username, placeholder: "Your pretty name here" %><br><br>
+
+  <%= form.label :email, "Email:" %><br>
+  <%= form.text_field :email, value: "example@example.com", type: "email" %><br><br>
```

Comment:
- [ ] The email field here should also use `email_field`. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_with.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))